src/api/public/apidocs-new/components/schemas/source/project_pattern.yaml

```
@@ -0,0 +1,39 @@
+type: object
+properties:
+  name:
+    type: string
+    xml:
+      attribute: true
+  rev:
+    type: string
+    xml:
+      attribute: true
+  vrev:
+    type: string
+    xml:
+      attribute: true
+  srcmd5:
+    type: string
+    xml:
+      attribute: true
+  entry:
+    type: object
+    properties:
```

```suggestion
  entry:
    type: array
    items:
      type: object
      properties:   
```

There can be more than one entry in the directory. This is an example of `_pattern` with more than one file:  https://build.opensuse.org/package/show/Cloud:OpenStack:Victoria/_pattern

---

Thanks for looking into it. It's updated